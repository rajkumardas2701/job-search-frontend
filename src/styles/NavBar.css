.navbar {
  display: flex;
  flex-direction: row;
  height: 50px;
  justify-content: space-between;
  position: fixed;
  left: 0;
  top: 0;
  width: 100%;
  justify-items: center;
  z-index: 90;
}

.logoHead {
  color: whitesmoke;
  width: 10%;
  text-align: center;
  font-family: 'Courier New', Courier, monospace;
  margin: 6px;
}

.logoAnc {
  margin: 10px 10px;
  width: 10%;
  text-decoration: none;
  height: 50px;
}

.logout-btn {
  width: 80px;
  height: 50px;
  border-radius: 5px;
  font-weight: 800;
}

.nav-btns {
  margin: 20px;
  width: 15%;
}

.nav-greet {
  margin: 25px auto;
  color: white;
  font-family: 'Courier New', Courier, monospace;
  font-weight: 800;
}

.logout-btn:hover {
  cursor: pointer !important;
  background-color: white;
}
