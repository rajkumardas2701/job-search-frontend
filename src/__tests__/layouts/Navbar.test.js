import React from 'react';
import renderer from 'react-test-renderer';
import { render, screen, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import NavBar from '../../layouts/Navbar';

afterEach(cleanup);

it('renders Navbar component correctly', () => {
  const navbar = renderer.create(<NavBar />).toJSON();
  expect(navbar).toMatchSnapshot();
});  
