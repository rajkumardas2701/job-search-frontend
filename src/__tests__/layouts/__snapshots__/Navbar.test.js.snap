// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Navbar component correctly 1`] = `
<div>
  <div
    className="navbar"
  >
    <div
      className="logo-section"
    >
      <a
        className="logoAnc"
        href="/"
      >
        <h1
          className="logoHead"
          data-testid="navbar-heading"
        >
          JobHub
        </h1>
      </a>
    </div>
    <div
      className="nav-greet"
    >
      
    </div>
    <div
      className="nav-btns"
    >
      <button
        className="logout-btn"
        onClick={[Function]}
        type="button"
      >
        Login
      </button>
    </div>
  </div>
</div>
`;
